packages/airnode-deployer/src/infrastructure/index.ts

```diff
@@ -333,6 +334,20 @@ export const deployAirnode = async (config: Config, configPath: string, secretsP
     const stageDirectory = getStageDirectory(directoryStructure, airnodeAddress, stage);
     if (stageDirectory) {
       logger.debug(`Deployment '${bucketStagePath}' already exists`);
+
+      const bucketMissingFiles = checkBucketMissingFiles(directoryStructure);
+      if (
+        bucketMissingFiles[airnodeAddress] &&
+        bucketMissingFiles[airnodeAddress][stage] &&
+        bucketMissingFiles[airnodeAddress][stage].length !== 0
+      ) {
+        throw new Error(
+          `Can't update an Airnode with missing files: ${bucketMissingFiles[airnodeAddress][stage].join(
+            ', '
+          )}. Deployer commands may fail and manual removal may be necessary.`
+        );
+      }
+
       const latestDeployment = Object.keys(stageDirectory.children).sort().reverse()[0];
       const bucketConfigPath = `${bucketStagePath}/${latestDeployment}/config.json`;
       logger.debug(`Fetching configuration file '${bucketConfigPath}'`);
```

Comment:
Can't select the correct line, it's meant to be the line below this one :smile: 

Even though the file will be present we should still wrap the call in `go` as it can throw for any other reason.
The same goes for JSON parsing.